### PR TITLE
resinCI: Add secrets and environment variables to .resinci.yml

### PR DESCRIPTION
**Pull request by brownjohnf**
Change-type: patch
Signed-off-by: Jack Brown <jack@resin.io>

### CODE REVIEW

File: .resinci.yml [hunk 1]
```diff
@@ -1,3 +1,17 @@
 ---
 docker:
-  publish: false
+  builds:
+    - dockerfile: Dockerfile
+      path: .
+      docker_repo: n/a
```
Comment (jviotti) [inline .resinci.yml]: Why is this `n/a`? Can we just omit it instead?

File: .resinci.yml [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+      publish: false
+      environment:
+        RESINOS_TESTS_APPLICATION_NAME: mochaTest
+        RESINOS_TESTS_DEVICE_TYPE: qemux86-64
+        RESINOS_TESTS_RESINOS_VERSION: *
+        RESINOS_TESTS_RESINIO_URL: resin.io
+        RESINOS_TESTS_RESINIO_STAGING_URL: resinstaging.io
+        RESINOS_TESTS_ENABLE_INTERACTIVE_TESTS: no
+      secrets:
+        - RESINOS_TESTS_API_KEY
```
Comment (jviotti) [inline .resinci.yml]: How is this populated? Are we hardcoding logic that populates this on the Docker pipeline?

Comment (horia-delicoti) [reply .resinci.yml]: @jviotti It will use https://github.com/resin-io/resin-vault/pull/1

**Comment by horia-delicoti** [comment]
[FD](https://www.flowdock.com/app/rulemotion/r-resinos/threads/F5EUJqRx7m65_zeqnck925ERhwz)

**Comment by telphan** [comment]
No longer relevant.

**Pull request closed without merging**